test/common/vmimages.py

```diff
@@ -116,7 +116,7 @@ def prune_images(force, dryrun):
         if os.path.isfile(path) and (path.endswith(".qcow2") or path.endswith(".partial")) and path not in targets:
             sys.stderr.write("Pruning {0}\n".format(filename))
             if not dryrun:
-                os.unlink(path)
+                os.unlink(filename)
```

Comment:
I disagree. `path` is the calculated, full path of the file to be checked. We don't want to delete something with the name `filename` somewhere, but the path that we just checked.